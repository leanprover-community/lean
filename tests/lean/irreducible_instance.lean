@[irreducible] instance : has_zero string := ⟨""⟩

class foo (s : string) : Prop
instance string.foo : foo 0 := ⟨⟩

#eval tactic.mk_instance `(foo "")