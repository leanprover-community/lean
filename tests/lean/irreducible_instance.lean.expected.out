irreducible_instance.lean:6:0: error: tactic.mk_instance failed to generate instance for
  foo ""
